### PR TITLE
clang-format 2016-03-29

### PR DESCRIPTION
**Pull request by mprobst**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### CODE REVIEW

File: Library/Formula/clang-format.rb [hunk 1]
```diff
@@ -1,15 +1,13 @@
 class ClangFormat < Formula
   desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
   homepage "http://clang.llvm.org/docs/ClangFormat.html"
-  version "2016-03-08"
+  version "2016-03-29"
 
   stable do
-    url "http://llvm.org/releases/3.8.0/llvm-3.8.0.src.tar.xz"
-    sha256 "555b028e9ee0f6445ff8f949ea10e9cd8be0d084840e21fbbe1d31d51fc06e46"
+    url "http://llvm.org/svn/llvm-project/llvm/tags/google/testing/2016-03-29/", :using => :svn
```
Comment (dunn) [inline Library/Formula/clang-format.rb]: Why the switch to SVN?

Comment (mprobst) [inline Library/Formula/clang-format.rb]: The `clang-format` formula was always using the tagged Google releases, the full LLVM releases are too infrequent (often several months apart). Somebody changed that in 20a0a92b39851b60eac8e03338585605e615c758, this just reverts back to the higher frequency releases.

Comment (dunn) [inline Library/Formula/clang-format.rb]: Got it, thanks!

**Comment by dunn** [comment]
Looks like it needs rebasing now, though.

**Comment by mprobst** [comment]
Sorry about that, done.

**Comment by dunn** [comment]
No problem! Merged as 44c923d6676440927816a409305c988320285ba2, thanks!

**Pull request closed without merging**